content/applications/productivity/data_cleaning.rst

```diff
@@ -211,16 +211,123 @@ Click on the :icon:`fa-clone` :guilabel:`Duplicates` smart button to manage thes
+specificity.
+
+.. tip::
+   The recycle rules run once every day, by default, as part of a scheduled action chron (*Data
+   Recycle: Clean Records*). However, each rule can be :ref:`run manually
+   <data-cleaning/run-recycle-rule>` anytime.
+
+By default, no recycle record rules exist. Select the :guilabel:`New` button to create a new rule.
+
+On the recycle record rule form, first choose a :guilabel:`Model` for this rule to target. Selecting
+a model updates the rule title to the chosen model.
+
+Optionally, configure a :guilabel:`Filter` to specify the records eligible for this rule. The number
+of eligible records is shown in the :icon:`oi-arrow-right` :guilabel:`# record(s)` link.
+
+Next, configure the field and time range for how the rule detects the records to recycle:
+
+- :guilabel:`Time Field`: select a field from the model to base the time (:dfn:`Delta`).
+- :guilabel:`Delta`: type the length of time, which must be a whole number (e.g. `7`).
+- :guilabel:`Delta Unit`: select the unit of time (:guilabel:`Days`, :guilabel:`Weeks`,
+  :guilabel:`Months`, or :guilabel:`Years`).
+
+Then, select a :guilabel:`Recycle Mode`:
+
+- :guilabel:`Manual`: requires each detected record to be manually recycled and enables the
+  :guilabel:`Notify Users` field.
+- :guilabel:`Automatic`: automatically merges recycles groupings, without notifying users.
+
+Lastly, select a :guilabel:`Recycle Action` to either :guilabel:`Archive` or :guilabel:`Delete`
+records. If :guilabel:`Delete` is selected, choose whether or not to :guilabel:`Include Archived`
+records in the rule.
+
+With the rule's configuration complete, either close the rule form, or :ref:`run the rule manually
+<data-cleaning/run-recycle-rule>` to instantly capture records to recycle.
+
+.. example::
+   A recycle rule can be configured to delete archived leads and opportunities that were last
+   updated a year ago and with a specific lost reason, by using the following configuration:
+
+   - :guilabel:`Model`: :guilabel:`Lead/Opportunity`
+   - :guilabel:`Filter`:
+
+     - `Active` `is` `not set`
+     - `Lost Reason` `is in` `Too expensive`
+
+   - :guilabel:`Time Field`: :guilabel:`Last Updated on (Lead/Opportunity)`
+   - :guilabel:`Delta`: `1`
+   - :guilabel:`Delta Unit`: :guilabel:`Years`
+   - :guilabel:`Recycle Mode`: :guilabel:`Automatic`
+   - :guilabel:`Recycle Action`: :guilabel:`Delete`
+   - :guilabel:`Include Archived`: :icon:`fa-check-square`
+
+   .. image:: data_cleaning/data-cleaning-recycle-rule.png
+      :align: center
+      :alt: Recycle records rule form for a lead/opportunity.
+
+.. _data-cleaning/run-recycle-rule:
+
+Manually run a recycle rule
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To manually run a specific recycle rule at any time, navigate to :menuselection:`Data Cleaning app
+--> Configuration --> Rules: Recycle Records`, and select the rule to run.
+
+Then, on the rule form, select the :guilabel:`Run Now` button on the top-left. Upon doing so, the
```

Comment:
```suggestion
Then, on the rule form, click the :guilabel:`Run Now` button on the top-left. Upon doing so, the
```
OPTIONAL: same reasoning as above re: "click" & "button" bias 😆 --- not required; just a suggestion.